hw_2_add.py

```diff
@@ -4,7 +4,9 @@ def smallest_integer(array):
     else:
         arr_max = max(array)
         return min(set(array).intersection(set(range(1,arr_max + 1)))) if arr_max > 0 else 1
-    
+
```

Comment:
this function works incorrectly. Pls check with examples in condition 
For example, given A = [1, 3, 6, 4, 1, 2], the function should return 5.
Given A = [1, 2, 3], the function should return 4.
Given A = [−1, −3], the function should return 1.

Reply:
binary_parser - also works incorrectly

Reply:
Thanks for review. All resolved.